catalog/plugin/catalog-plugin-metacardbackup-s3storage/src/main/java/org/codice/ddf/catalog/plugin/metacard/backup/storage/s3storage/MetacardS3StorageRoute.java

```diff
@@ -88,11 +89,9 @@
 
   public MetacardS3StorageRoute(CamelContext camelContext) {
     super(camelContext);
-    registry = new org.apache.camel.impl.SimpleRegistry();
-    CompositeRegistry compositeRegistry = new CompositeRegistry();
-    compositeRegistry.addRegistry(camelContext.getRegistry());
-    compositeRegistry.addRegistry(registry);
-    ((DefaultCamelContext) camelContext).setRegistry(compositeRegistry);
```

Comment:
See https://camel.apache.org/manual/latest/camel-3-migration-guide.html#_generic_information

CompositeRegistry is removed. DefaultRegistry is recommended replacement